src/notifications/android_setup.md

```diff
@@ -0,0 +1,7 @@
+# Android
+
+In order to add the Notification Plugin to your Android application, first you need to setup the Foreground Service, then add the Notification Plugin dependency and integrate it:
+
+## Next steps
+- **[Setting up the Foreground Service](android_service.md)**
```

Comment:
⚠️ Exemplary suggestion.

If previous suggestion to use title cases on titles is to be implemented, it should affect all other links and the titles on the pages themselves.
```suggestion
- **[Setting Up the Foreground Service](android_service.md)**
```